superset/connectors/sqla/models.py

```diff
@@ -84,6 +84,7 @@ class AnnotationDatasource(BaseDatasource):
     """
 
     cache_timeout = 0
+    changed_on = None
```

Comment:
I would rather place this on `BaseDatasource` in `superset/connectors/base/models.py`, so as to make sure it propagates to all child classes.

Reply:
@villebro I'm hesitant to do that as changed_on is a sa.Column and is only causing problems for the classes that inherit from AuditMixinNullable and are not actual sqllachemy models if I understand it right. AnnotationDatasource is an exception case there.
```
    changed_on = sa.Column(
        sa.DateTime, default=datetime.now, onupdate=datetime.now, nullable=True
    )
```